packages/slice-machine/tests/mocks.test.js

```diff
@@ -1,16 +1,315 @@
-import * as Widgets from "@lib/models/common/widgets";
-import { handleFields } from "@lib/mock/misc/handlers";
-import { model, createExpector } from "./__mockData__/models";
+import * as Widgets from "../lib/models/common/widgets";
+import { handleFields } from "../lib/mock/misc/handlers";
 
-const mockFields = handleFields(Widgets);
+const toHaveProps = (obj, props) =>
+  props.forEach((prop) => expect(obj).toHaveProperty(prop));
 
-test("Can create mocks", () => {
-  const primary = Object.entries(model.variations[0].primary);
-  const mockEntries = Object.entries(mockFields(primary));
-  expect(mockEntries.length).toBe(primary.length);
+describe("handleFields", () => {
+  const mockFields = handleFields(Widgets);
+  test("can create a text mock", () => {
+    const textField = [
+      [
```

Comment:
I would use the constructor of the widget directly

Reply:
Good idea :)